graphql/resolvers.go

```diff
@@ -2689,6 +2689,29 @@ func (r *queryResolver) BbGetCreatedTickets(ctx context.Context, taskID string)
 	return createdTickets, nil
 }
 
+func (r *queryResolver) BuildVariantHistory(ctx context.Context, projectId string, buildVariant string) ([]string, error) {
+	buildVariantTasks, err := task.FindTaskNamesByBuildVariant(projectId, buildVariant)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error while getting tasks for '%s': %s", buildVariant, err.Error()))
+	}
+	if buildVariantTasks == nil {
```

Comment:
Should this be an error or just an empty list? Not sure when this is used but theoretically if a user typoed something I don't think we'd want to show them an error

Reply:
good catch